contracts/token/ERC1155/IERC1155.sol

```diff
@@ -10,6 +10,11 @@ import {IERC165} from "../../utils/introspection/IERC165.sol";
  * https://eips.ethereum.org/EIPS/eip-1155[ERC].
  */
 interface IERC1155 is IERC165 {
+    /**
+     * @dev Emitted when zero amount of token transferred from `from` to `to` by `operator` with safeBatchTransferFrom.
+     */
+    event TransferZero(address indexed operator, address indexed from, address indexed to);
+    
```

Comment:
This is not part of the ERC-1155 standard. Even if we want it (I don't think we do) it should no be here.